lib/pre-build.js

```diff
@@ -186,7 +187,7 @@ function execGyp(args, cb) {
 }
 
 function build(target, rebuild, cb) {
-  if (IS_WIN) {
+  if (IS_WIN && HAS_NEW_NODE_GYP_ARGS) {
```

Comment:
I believe this logic should be negated (`&& !HAS_NEW_NODE_GYP_ARGS`)? On the newer versions of gyp we do not want to prepend `/t:`, but older versioned still need it.